lib/config.yml

```diff
@@ -263,7 +266,7 @@ RSpec/NoExpectationExample:
   Enabled: false
 
 RSpec/NotToNot:
-  EnforcedStyle: to_not
+  Enabled: false
```

Comment:
This is probably my least favorite rule in rubocop. It makes no difference to anything and no matter how many times it catches me I still second-guess myself every time.